JapaneseFilter/sections/whitelist.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! White list. Fixing filtration errors(false-positive)
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/64912
+@@||ranking.rakuten.co.jp^$domain=bookmark.rakuten.co.jp
```

Comment:
I see in AdAway:
`127.0.0.1 api.ranking.rakuten.co.jp`
`127.0.0.1 shopapi.ranking.rakuten.co.jp`
Which URL is blocked?

Reply:
The latter:

![rakuten2](https://user-images.githubusercontent.com/58900598/94915189-64cc5000-04e7-11eb-955b-8eb25312533d.png)

I have never used AdAway.

Reply:
You beat me, but can you add
`||rakuten.co.jp/com/img/thumb/logout/bnr/`?

For this:

![rakuten](https://user-images.githubusercontent.com/58900598/94915292-96451b80-04e7-11eb-8cbf-6b8b0f170f6a.png)